helper-scripts/azure-prep-image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+#
+# Create an image from a page blob and copy it to a custom region (so as to
+# cirumvent regional CPU limitationf when running tests)
+#
+
+function disk_from_page_blob() {
+	local name="$1"
+	local rg="$2"
+
+	local out=$(mktemp)
+	trap "rm -f $out" EXIT
+
+	az disk create --name "$name" -g "$rg" \
+		--source https://$rg.blob.core.windows.net/publish/"$name".vhd \
+		> "$out"
+	
+	# print disk id
+	cat "$out" | jq -r '.id'
+
+	rm -f "$out"
+	trap - EXIT
+}
+# --
+
+function image_from_disk() {
+	local name="$1"
+	local rg="$2"
+	local disk_id="$3"
+
+	az image create -g $rg --name "$name" --source "$disk_id" --os-type linux
+}
+# --
+
+function copy_to_region() {
+	local name="$1"
+	local rg="$2"
+	local dest_rg="$3"
+	local dest_region="$4"
+
+	az image copy --source-resource-group $rg --source-object-name "$name" \
+		--target-location "$dest_region" --target-resource-group "$dest_rg" \
+		--cleanup
+}
+# --
+
+function main() {
+	local rg="flatcar"
+	local dest_rg="flatcar-release-testing-eu-n"
+	local dest_region="northeurope"
+
+	local blob_name="$1"
+	[ -z "$blob_name" ] && {
+			echo
+			echo "Usage: $0 <page-blob-namei-without-vhd-suffix>"
+			echo ""
+			echo "		e.g."
+			echo "		$0 flatcar-linux-2430.0.0-alpha"
+			echo ""
+			exit 1
+	}
+
+	echo "Creating VM disk image from page blob $blob_name in resource group"
+	echo " $rg and storing it in rg $dest_rg in region $dest_region."
+	echo
+	
+	echo " #### 1. Creating disk from page blob"
+	local disk_id=$(disk_from_page_blob "$blob_name" "$rg")
+	echo " ----> Generated disk with ID $disk_id"
+	
+	echo " #### 2. Creating image from disk"
+	image_from_disk "$blob_name" "$rg" "$disk_id"
+
+	echo " #### 3. Copying image to rg $dest_rg in region $dest_region"
+	copy_to_region "$blob_name" "$rg" "$dest_rg" "$dest_region"
+
+	echo " ====> All done."
+}
+# --
+
+if [ "$(basename $0)" = "prep-image.sh" ] ; then
```

Comment:
d/prep-image/azure-prep-image/

Reply:
Legacy names from earlier during script development, meh. Thank you, good catch!